java/flight/flight-sql-jdbc-core/src/test/java/org/apache/arrow/driver/jdbc/utils/MockFlightSqlProducer.java

```diff
@@ -363,6 +364,15 @@ public Runnable acceptPutStatement(
     };
   }
 
+  @Override
+  public Runnable acceptPutStatementBulkIngest(
+      final CommandStatementIngest commandStatementIngest,
+      final CallContext callContext,
+      final FlightStream flightStream,
+      final StreamListener<PutResult> streamListener) {
+    throw new UnsupportedOperationException();
```

Comment:
This is in jdbc. Since jdbc does not support bulk ingestion, can you please suggest what can be done in the mock? Do you mean return 0 rows updated as PutResult?

Reply:
Done. Included new commit with this change